nixos/modules/virtualisation/docker-containers.nix

```diff
@@ -161,6 +161,15 @@ let
             ["--network=host"]
           '';
         };
+        restart = mkOption {
+          type = types.enum [ "no" "on-success" "on-failure" "on-abnormal" "on-watchdog" "on-abort" "always" ];
```

Comment:
Does `docker run` implement the systemd watchdog pinger?

Reply:
I don't think it does currently but a user might patch their docker with [this](https://github.com/moby/moby/pull/13750) and I don't see much reason to leave it out either.